client/client.go

```diff
@@ -108,7 +108,7 @@ func GetClient(clientURL, username, password, proxy, proxyCreds string, rateLimi
 }
 
 //MakeRequest generate the HTTP request with required headers
-func (client *Client) MakeRequest(method, path string, body *container.Container, bodyBytes []byte, authenticated bool) (*http.Request, error) {
+func (client *Client) MakeRequest(method, path string, body *container.Container, bodyBytes []byte, bodyBuffer *bytes.Buffer, authenticated bool) (*http.Request, error) {
```

Comment:
Cannot we make single parameter for bodyBytes and bodyBuffer? If yes, then please do it.

Reply:
Merged Successfully